### ISSUE TITLE
Dev test: Accessible John Travoltage 1.3.0-dev.20

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#831]
@ariel-phet @emily-phet @phet-steele Given the number of description changes from the last dev version, we should do one more round of dev testing before moving to an RC with accessibility features, which will also make the RC process faster.

Link to Sim: http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.36/john-travoltage_en.html

The only changes have been for descriptions and screen reader behavior, so please just test the following platforms:

 - [x] JAWS + Latest version of IE11
 - [x] NVDA + Latest version of Firefox
 - [x] VoiceOver + Safari in latest version of MacOS

In addition to testing tab navigation and using sim elements with a screen reader, please make sure that all of the content in this document can be read by the assistive technology: http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.36/john-travoltage-a11y-view.html

If needed, the following includes quick references for how to read web content with a screen reader:

 - http://webaim.org/articles/jaws/
 - http://webaim.org/articles/nvda/
 - http://webaim.org/articles/voiceover/

If you would like to practice reading content, try reading a simple HTML page like this one: http://csb.stanford.edu/class/public/pages/sykes_webdesign/05_simple.html

Other potentially helpful query parameters:
?valueText: Shows the description of the arm and leg that the screen reader should read when their position changes.

Please let me know if you have any questions about the above test! Assigning to @ariel-phet to prioritize and @emily-phet so that she is aware of this task.

**Comment by emily-phet** [comment]
@ariel-phet - just to give you an idea of timeline - keyboard and screen reader accessible version of JT needs to be published to PhET website by start of NSF Video Showcase (May 15th).

**Comment by jessegreenberg** [comment]
@phet-steele sorry, I should have also mentioned, we are aware of VoiceOver and JAWS bugs which were found in
 https://github.com/phetsims/john-travoltage/issues/237 and https://github.com/phetsims/john-travoltage/issues/227, which have both been marked as ```wontfix```.

**Comment by ariel-phet** [comment]
Given the very short timeline, marking this as top priority

**Comment by phet-steele** [comment]
@jessegreenberg @emily-phet I believe testing here is done. rc up next!

**Comment by emily-phet** [comment]
@jessegreenberg @phet-steele Great!

**Comment by jessegreenberg** [comment]
Excellent, thanks @phet-steele!  https://github.com/phetsims/john-travoltage/issues/243 will be closed shortly, and remaining open issues have been investigated and marked as ```wontfix```.  RC will be ready shortly.
